scadalts-ui/src/apps/App.vue

```diff
@@ -80,7 +80,7 @@
 							<span>Edit profile</span>
 							<v-icon>mdi-account-box</v-icon>
 						</v-btn>
-						<v-btn block text link href="./logout.htm">
+						<v-btn block text link @click="logout()">
```

Comment:
I suggest you use "v-on:click"

Reply:
https://vuejs.org/v2/style-guide/#Directive-shorthands-strongly-recommended
Based on that article:
> Directive shorthands (: for v-bind:, @ for v-on: and # for v-slot) should be used always or never.

I am using this shorthands in a consistent way inside whole project, which means that I am using them 'always'.